ci/openshift-ci/peer-pods-azure.sh

```diff
@@ -0,0 +1,213 @@
+#!/bin/bash -e
+# Setup peer-pods using cloud-api-adaptor on azure
+
+###############################
+# Disable security to allow e2e
+###############################
+
+# Disable security
+oc adm policy add-scc-to-group privileged system:authenticated system:serviceaccounts
+oc adm policy add-scc-to-group anyuid system:authenticated system:serviceaccounts
+oc label --overwrite ns default pod-security.kubernetes.io/enforce=privileged pod-security.kubernetes.io/warn=baseline pod-security.kubernetes.io/audit=baseline
+
+
+####################################
+# Get basic credentials from cluster
+####################################
+
+oc -n kube-system get secret azure-credentials -o json > azure_credentials.json
+AZURE_CLIENT_ID="$(jq -r .data.azure_client_id azure_credentials.json|base64 -d)"
+AZURE_CLIENT_SECRET="$(jq -r .data.azure_client_secret azure_credentials.json|base64 -d)"
+AZURE_TENANT_ID="$(jq -r .data.azure_tenant_id azure_credentials.json|base64 -d)"
+AZURE_SUBSCRIPTION_ID="$(jq -r .data.azure_subscription_id azure_credentials.json|base64 -d)"
```

Comment:
It seems that `azure_credentials.json` isn't needed anymore after this. It should be deleted.

Reply:
ack